src/data/equiv/basic.lean

```diff
@@ -512,6 +512,18 @@ def arrow_punit_equiv_punit (α : Sort*) : (α → punit.{v}) ≃ punit.{w} :=
 ⟨λ f, punit.star, λ u f, punit.star,
   λ f, by { funext x, cases f x, refl }, λ u, by { cases u, reflexivity }⟩
 
+/-- If `α` is `subsingleton` and `a : α`, then the type of dependent functions `Π (i : α), f
+i` is equivalent to `f i`. -/
+def Pi_subsingleton (α) (f : α → Sort*) [subsingleton α] (a : α): (Π (i : α), f i) ≃ f a :=
```

Comment:
```suggestion
def Pi_subsingleton {α} (β : α → Sort*) [subsingleton α] (a : α) : (Π a', β a') ≃ β a :=
```
is more in line with how we usually name type families. Note that `{α}` can be implicit as it's obvious from `a`.